test/studies/hpcc/HPL/vass/hpl.hpcc2012.chpl

```diff
@@ -13,7 +13,7 @@ use LinearAlgebra, Random, Time;
 // Use the distributions we need for this computation
 //
 use DimensionalDist2D, ReplicatedDim, BlockCycDim;
-use ReplicatedDist;
+use ReplicatedDist, ChplConfig;
```

Comment:
I'm not seeing why this was imported here. Had `ReplicatedDist` previously been importing `ChplConfig` with it?